CommonApi/src/main/java/mezz/jei/api/registration/IVanillaCategoryExtensionRegistration.java

```diff
@@ -13,6 +14,10 @@
  * {@link IModPlugin#registerVanillaCategoryExtensions(IVanillaCategoryExtensionRegistration)}
  */
 public interface IVanillaCategoryExtensionRegistration {
+	/**
+	 * {@link IJeiHelpers} provides helpers and tools for addon mods.
+	 */
```

Comment:
Can you please bump the `specificationVersion` number in `gradle.properties` to `12.4.0`
https://github.com/mezz/JustEnoughItems/blob/c6c115cfad74659c56cfbc293992e064f9deae0f/gradle.properties#L41

and write `@since 12.4.0` in the javadoc here so that people know which version of JEI to depend on if they need this feature?